backend/models/event.py

```diff
@@ -0,0 +1,22 @@
+from pydantic import BaseModel
+from datetime import datetime
+
+__authors__ = ["Ajay Gandecha", "Jade Keegan", "Brianna Ta", "Audrey Toney"]
+__copyright__ = "Copyright 2023"
+__license__ = "MIT"
+
+class Event(BaseModel):
+    """
+    Pydantic model to represent an `Event`.
+    
+    This model is based on the `EventEntity` model, which defines the shape
+    of the `Event` database in the PostgreSQL database
+    """
+    
+    id: int | None=None
+    name: str
+    time: datetime
+    location: str
+    description: str
+    public: bool
+    organization_id: int
```

Comment:
I'm not certain this makes sense to have in the `Event` model. This feels beneath the model level of abstraction.

Reply:
This is required in order to post events! I originally created a `NewEvent` model for this purpose but it was removed later in the testing branch.